### ISSUE TITLE
log location

### ISSUE THREAD

**Issue by bigmac5753** [kubec/chia-log-analyzer#11]
In linux, opening the file in the same location as a log file always opens up the log in the default location.

Also, the --log= option doesn't work, I get this error:

```
mike@linux-mm:~/.chia/mainnet/log$ ./chia-log-analyzer.go-linux-amd64 --log=/home/mike/.seno2/mainnet/log/debug.log
flag provided but not defined: -log
Usage of ./chia-log-analyzer.go-linux-amd64:
  -writelog
    	write log?
```

**Comment by kubec** [comment]
Hello, thank you for report. I will try print more info when cannot open file.

**Comment by H34dl3ss** [comment]
I moved the .chia folder to another location with the CHIA_ROOT environment variable.
The binary is still in the log directory but starting with absolute path is not working anymore.

```
 /new/path/.chia/mainnet/log/chia-log-analyzer.go-linux-arm
Please specify path to the log file, with parameter: log (--log=/path/to/debug.log)
```

```
/new/path/.chia/mainnet/log/chia-log-analyzer.go-linux-arm --log=/new/path/.chia/mainnet/log/debug.log
flag provided but not defined: -log
Usage of /media/Daten/.chia/mainnet/log/chia-log-analyzer.go-linux-arm:
  -writelog
        write log?
```

It is working if I cd into the log directory and start the binary from that location.

**Comment by kubec** [comment]
Fixed in new release 0.5
